client/snapshot.go

```diff
@@ -197,6 +197,30 @@ func (client *Client) SnapshotExport(setID uint64) (stream io.ReadCloser, conten
 		}
 		return nil, 0, fmt.Errorf("unexpected status code: %v", rsp.Status)
 	}
+	contentType := rsp.Header.Get("Content-Type")
+	if contentType != "application/x.snapd.snapshot" {
+		return nil, 0, fmt.Errorf("cannot import snapshot of content type %q", contentType)
+	}
 
 	return rsp.Body, rsp.ContentLength, nil
 }
+
+// SnapshotImportSet is a snapshot import created by a "snap import-snapshot".
+type SnapshotImportSet struct {
+	ID    uint64   `json:"set-id"`
+	Snaps []string `json:"snaps"`
+}
+
+// SnapshotImport imports an exported snapshot set.
+func (client *Client) SnapshotImport(exportStream io.Reader) (SnapshotImportSet, error) {
+	headers := map[string]string{
+		"Content-Type": "application/x.snapd.snapshot",
```

Comment:
This should be a const somewhere.

Reply:
Happy to do it like e.g. this https://paste.ubuntu.com/p/7TZKPfpxd3/ - I'm inclined to do it as a followup as I'm a bit worried this becomes a bikeshed for this PR. But maybe I'm too pesisimtic :)

Reply:
As long as it ends up a const ;)